templates/node/index.d.ts.twig

```diff
@@ -167,13 +167,36 @@ declare module "{{ language.params.npmPackage|caseDash }}" {
     
     static limit(value: number): string;
     
-    static offset = (value: number): string;
+    static offset(value: number): string;
 
     private static addQuery(attribute: string, method: string, value: QueryTypes): string;
 
     private static parseValues(value: QueryTypes): string;
   }
 
+  export class ID {
+    static unique(): string;
+    static custom(id: string): string;
+  }
+
+  export class Permission {
+    static read(role: string): string;
+    static write(role: string): string;
+    static create(role: string): string;
+    static update(role: string): string;
+    static delete(role: string): string;
+  }
+
+  export class Role {
+    static any(): string;
+    static user(id: string, status?: string): string;
+    static users(status?: string): string;
+    static guests(): string;
+    static team(id: string, role?: string): string;
+    static member(id: string): string;
+    static status(status: string): string;
```

Comment:
This should not be here as the functionality is now part of the `user` and `users` roles as the `status` parameter; corresponding PR to remove the methods is here: https://github.com/appwrite/sdk-generator/pull/554

Reply:
Removed